docs/running-on-mesos.md

```diff
@@ -406,6 +406,22 @@ See the [configuration page](configuration.html) for information on Spark config
     If unset it will point to Spark's internal web UI.
   </td>
 </tr>
+<tr>
+  <td><code>spark.mesos.rejectOfferDurationForUnmetConstraints</code></td>
+  <td><code>120s</code></td>
+  <td>
+    Set the amount of time for which offers are rejected when constraints are unmet. See <code>spark.mesos.constraints</code>.
+    This is used to prevent starvation of other frameworks.
+  </td>
+</tr>
+<tr>
+  <td><code>spark.mesos.rejectOfferDurationForReachedMaxCores</code></td>
```

Comment:
I would actually not document these configs. Doing so would require us to maintain backward compatibility. I can't think of any strong use case where someone would want to change these values so I don't think it's worth the maintenance burden.